### DISCUSSION TITLE
Having difficulty installing Puddletag on OpenSUSE (Tumbleweed)

### DISCUSSION THREAD

**Discussion by thepiecesofme** [puddletag/puddletag#584] [Q&A] [answered]
Hi,

I've tried other tagging apps but nothing comes close to Puddletag. I used it on a different distro previously but now I use OpenSUSE Tumbleweed and I'm having difficulty.

I tried to install the rpm from another distro. That didn't work because it couldn't find pyqt5, despite it being on my system. I tried the snap (I think we're up to 3-4 times now). Because snaps are sandboxed, I can't actually access any of my media files on my external HDDs to tag. I tried manually setting the interface in the snap (I literally walked through every single possibility to find the right one) but that failed, because it appears that the option has to be included in the setup when creating the snap. I uninstalled and tried a few more times to install the snap in "classic" mode (which gives access to most of the system). Unfortunately, this too failed, though I'm less sure why. I think maybe classic mode also needs to be setup as a possible option when the snap is being packaged.

I tried to install alien to convert the deb to rpm, but that's a work in progress. I think it's installed but since I couldn't get the actual rpm to install due to the pyqt5 issue, I don't think that alien will help. I'm not even sure if I could compile from source (which I don't yet know how to do), if that would make any difference, since I'd still have the pyqt5 issue.

Please know, none of this is complaints. I can't imagine that I'm the only one with media to be tagged on external HDDs and if I'm running into these issues, others may be, too. I've posted on multiple other forums from Reddit, to OpenSUSE, to LinuxQuestions but there were few suggestions and the ones that were shared didn't work because of the need to have the options within the snap. I'm just wondering if anyone has any thoughts on how I could succeed on this. Whether a way to get pyqt5 on my system so it's found, or perhaps adding the --classic flag as an option in the next snap package, or something else altogether. I just really want to get Puddletag working. :) I hope this is an appropriate place to posit these questions. I looked for contact info but didn't find any and I'm kind of getting desperate since I'm running out of places to search for help. Thanks so much!

**Comment by ghost** [comment, marked as answer]
At the moment, OpenSUSE Tumbleweed doesn't have a package for puddletag, the one they have for Leap is still on the old version (Python2 version, the current one has been ported to Python3 and PyQt5).

You could try downloading the latest release from https://github.com/puddletag/puddletag/releases, or clone the repository from Github, and then install the dependencies as detailed in the README. You can get the dependencies using pip (except for Chromaprint) so you can use puddletag in a virtual environment.

I haven't tried accessing music on an external drive using puddletag, but I don't have any problems running it on local files. I run puddletag on Tumbleweed from a git clone as described above.

For puddletag support/bugs you should use the issue tracker on Github, this is where the development is happening now.
